types/dragino/lds02/default.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$id": "https://akenza.io/dragino/lds02/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "open": {
+      "title": "Open",
+      "description": "Status of the magnet. open = true, closed = false",
+      "type": "boolean"
+    },
+    "waterleak": {
+      "title": "Waterleak",
+      "description": "Equals true if a waterleak is detected",
+      "type": "boolean"
+    },
+    "alarm": {
+      "title": "Alarm",
+      "description": "Equals true if the alarm is triggered",
+      "type": "boolean"
+    },
+    "openTimes": {
```

Comment:
possibly rename this to openCount? or is this name set by the manufacturer?

Reply:
sure yes but lets change it